markdown/m2h/index.js

```diff
@@ -6,32 +6,36 @@ const gfm = require("remark-gfm");
 const raw = require("rehype-raw");
 const format = require("rehype-format");
 
-const handlers = require("./handlers");
+const buildLocalizedHandlers = require("./handlers");
 const { decodeKS, encodeKS } = require("../utils");
 
-function makeProcessor() {
+function makeProcessor(options) {
+  const localizedHandlers = buildLocalizedHandlers(options.locale);
```

Comment:
I wasn't able to find a more elegant way to convey the option/locale to the handler so I went with "building" them here.
Let me know if you think of something which could be better.